src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav__list {
```

Comment:
```suggestion
.nav {
  &__list {
```

Reply:
you still can use it like this even if you don`t need any style properties on nav